service/src/test/java/bio/terra/externalcreds/TestUtils.java

```diff
@@ -20,9 +20,13 @@ public static Timestamp getRandomTimestamp() {
   }
 
   public static LinkedAccount createRandomLinkedAccount() {
+    return createRandomLinkedAccount("ras");
```

Comment:
I want to change the provider param from a string type to an enum and replace this hard-coded value: https://github.com/DataBiosphere/terra-external-credentials-manager/blob/dev/common/openapi.yml#L247

Reply:
That sounds like an excellent idea!